build.sbt

```diff
@@ -1,32 +1,76 @@
-name := "better-monadic-for"
-organization := "com.olegpy"
-version := "0.2.4"
-licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
-homepage := Some(url("http://github.com/oleg-py/better-monadic-for"))
+lazy val scala213 = "2.13.0-RC1"
+lazy val scala212 = "2.12.8"
+lazy val scala211 = "2.11.12"
+lazy val supportedScalaVersions = List(scala211, scala212, scala213)
 
-scalaVersion := "2.12.5"
-crossScalaVersions := Seq("2.11.12", "2.12.5")
-libraryDependencies ++= Seq(
-  scalaOrganization.value % "scala-compiler" % scalaVersion.value,
-  "org.scalatest" %% "scalatest" % "3.0.5" % Test,
-  "io.monix" %% "monix-eval" % "3.0.0-RC1" % Test,
-  "org.ensime" %% "pcplod" % "1.2.1" % Test,
-  "org.scalaz" %% "scalaz-core" % "7.2.20" % Test,
+ThisBuild / organization := "com.olegpy"
+ThisBuild / version := "0.2.4"
+ThisBuild / licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
+ThisBuild / homepage := Some(url("http://github.com/oleg-py/better-monadic-for"))
+ThisBuild / scalaVersion := scala212
+
+val testSettings = Seq(
+  libraryDependencies ++= Seq(
+    "org.scalatest" %% "scalatest" % "3.0.8-RC2" % Test
```

Comment:
Why migrate to scalatest? Looks like there's a [version of utest](https://mvnrepository.com/artifact/com.lihaoyi/utest) for RC1

Reply:
Huh, guess I derped on that one 😅